src/index.css

```
@@ -8,14 +8,14 @@
-  @apply flex justify-center w-full min-h-screen bg-white;
-    font-size: 62.5%;
-    display: flex;
-    font-size: 62.5%;
-    justify-content: center;
-    min-height: 100vh;
-    margin: 0 auto;
-    font-family: 'Pretendard';
+  @apply flex min-h-screen w-full justify-center bg-white;
+  font-size: 62.5%;
+  display: flex;
+  font-size: 62.5%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)